File: Back/src/lib/database/models/game.js
'use strict';
const {
  Model
} = require('sequelize');
module.exports = (sequelize, DataTypes) => {
  class Game extends Model {
    /**
     * Helper method for defining associations.
     * This method is not a part of Sequelize lifecycle.
     * The `models/index` file will call this method automatically.
     */
    static associate(models) {
      // define association here
    }
  }
  Game.init({
    c_id: DataTypes.INTEGER,
    d_id: DataTypes.INTEGER,
    num_arr: DataTypes.TEXT,
    c_score: DataTypes.INTEGER,
    d_score: DataTypes.INTEGER
  }, {
    sequelize,
    modelName: 'Game',
  });
  return Game;
};

File: Front/app/src/main/java/com/madcamp/project2/Data/Game.kt
package com.madcamp.project2.Data

data class Game(
    val id: Int,
    val c_id: Int,
    val d_id: Int,
    val num_arr: String,
    val c_score: Int,
    val d_score: Int
)

data class RequestGame (
    val c_id: Int,
    val d_id: Int,
    val num_arr: String
    )
